projects/canopy/src/lib/forms/radio/radio-button--segment.component.scss

```diff
@@ -51,9 +52,19 @@
       cursor: not-allowed;
     }
 
+    &:hover + .lg-radio-button__label {
+      background-color: var(--color-super-blue-dark);
```

Comment:
@AlexandruHoisan instead of hard-coding the blue here, could you please create a new variable in `_radio.scss` called `--radio-segment-hover-bg-color: var(--color-super-blue-dark)` and then refer to that please?

This allows teams to do theming in a better/generic way.

Reply:
Nice One Elena. Yep, makes sense. Done :)